lib/ble.toit

```diff
@@ -93,13 +100,15 @@ class Advertiser:
   The data is advertised for the given $duration and once every $interval.
     If the $duration is null, then the data is advertised indefinitely.
 
+  The advertise will include the given connection_mode, use one of the BLE_CONNECTION_MODE_* constants
+
   Use $set_data to set the data.
 
   Only one advertiser can advertise at any given time.
   */
-  start --duration/Duration?=null --interval/Duration=DEFAULT_INTERVAL:
+  start --duration/Duration?=null --interval/Duration=DEFAULT_INTERVAL --connection_mode/int=BLE_CONNECT_MODE_NONE:
```

Comment:
Consider putting the long parameters on individual lines. Maybe add a return type in the process.

```
start -> none
    --interval/Duration=DEFAULT_INTERVAL
    --connection_mode/int=BLE_CONNECT_MODE_NONE:
  ...
```